packages/bot-web-ui/src/components/dashboard/bot-builder/bot-builder.tsx

```diff
@@ -22,6 +21,22 @@ const BotBuilder = observer(() => {
     const { onMount, onUnmount } = app;
     const el_ref = React.useRef<HTMLInputElement | null>(null);
 
+    const trackRudderStackForBotBuilder = (action: TActions) => {
+        RudderStack.track('ce_bot_builder_form', {
+            action,
+            form_source: 'ce_bot_builder_form',
+        });
+    };
+
+    React.useEffect(() => {
+        if (active_tab === DBOT_TABS.BOT_BUILDER) {
+            trackRudderStackForBotBuilder('open');
+            return () => {
+                trackRudderStackForBotBuilder('close');
+            };
+        }
+    }, [active_tab]);
```

Comment:
@NikitK-deriv regarding this change we just need to send the action that we are performing and form source 
because we handing the account_type from the main implementation itself.
will add device_type .

Reply:
valid point, thanks for explanation 👍 It's schema issue, we'll change that